jekyll/_cci2/language-python.md

```diff
@@ -55,27 +51,16 @@ The Python [orb]({{site.devhub_base_url}}/orbs/orb/circleci/python) contains a s
 To add the orb to your config, insert:
 ```yaml
 orbs:
-  python: circleci/python@1.5.0
-```
-
-Note: You might need to enable organization settings to allow the use of third-party orbs in the CircleCI dashboard, or request permission from your organization’s CircleCI admin.
-
-### 3. Create a workflow
-{: #create-a-workflow }
-
-A workflow is a set of rules for defining a collection of jobs and their run order. Workflows support complex job orchestration using a set of configuration keys to help you resolve failures sooner. Inside the workflow, you define the jobs you want to run. CircleCI will run this workflow on every commit. Learn more about [workflow configuration]({{ site.baseurl }}/2.0/configuration-reference/#workflows).
-
-```yaml
-workflows:
-  my_workflow: # This is the name of the workflow, feel free to change it to better match your workflow.
+  python: circleci/python@2.0.3
 ```
+**Note**: It is always a good idea to check in the [Orb Registry](https://circleci.com/developer/orbs) to ensure you are using the most recent (or correct for your project) orb verison.
 
-### 4. Create a job
+### 3. Create a job
 {: #create-a-job }
 
-Jobs are the building blocks of your config. Jobs are collections of steps, which run commands/scripts as required. All of the steps in the job are executed in a single unit, either within a fresh container or Virtual Machine. Learn more about [jobs]({{site.baseurl}}/2.0/configuration-reference/#jobs).
+Jobs are the building blocks of your config. Jobs are collections of steps, which run commands/scripts as required. All of the steps in the job are executed in a single unit, either within a fresh container or Virtual Machine. Learn more about jobs on the [Jobs and Steps]({{site.baseurl}}/2.0/jobs-steps/) page.
 
-A traditional ask from developers who are getting started with CircleCI is to perform 3 basic tasks: `build`, `test` and `deploy`. This section will guide you through each of the config changes needed. Because we are using the official Python orb, these steps can easily be accomplished:
+A common ask from developers who are getting started with CircleCI is to perform 3 basic tasks: `build`, `test` and `deploy`. This section guides you through each of the config changes needed. Because we are using the official Python orb, we can use commands that are built into the orb to keep our config simple and succinct:
```

Comment:
```suggestion
A common ask from developers who are getting started with CircleCI is to perform three basic tasks: `build`, `test` and `deploy`. This section guides you through each of the config changes needed. Because we are using the official Python orb, we can use commands that are built into the orb to keep our config simple and succinct:
```